test/e2e/lib.ts

```diff
@@ -1,14 +1,14 @@
 class MyClass {
-  field: string;
+  _field: string;
```

Comment:
Unsupported node type PrivateKeyword
maybe you just mean later when you merge some more changes